app/assets/stylesheets/scaffolds.scss

```diff
@@ -189,11 +196,13 @@ main {
   }
 }
 
+.search-results {
+  text-align: center;
+}
+
 .user-image {
-  img {
-    max-width: 100px;
-    margin: 0 0 15px;
-  }
+  max-width: 100px;
+  margin: 0 0 15px;
 }
```

Comment:
Changes to `.user-image` class removed `img` width constraints. Element should be constrained to 100px for proper scaling of `/images/missing.png`

See http://imgur.com/a/6t53E